client/site-profiler/components/domain-information/verified-provider.tsx

```diff
@@ -22,9 +23,15 @@ export default function VerifiedProvider( props: Props ) {
 				{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
 				<Gridicon icon="checkmark" size={ 10 } />
 			</span>
-			<a href={ hostingProviderHomepage }>{ hostingProviderName }</a>
-			&nbsp;&nbsp;
-			<a href={ hostingProviderLogin }>({ translate( 'login' ) })</a>
+			<a href={ showHosting ? 'https://wordpress.com' : hostingProviderHomepage }>
+				{ showHosting ? translate( 'WordPress.com' ) : hostingProviderName }
```

Comment:
The naming `showHosting` confused me a little, if this means "showing hosting provider" it seems the condition there needs to be the other way around.  I was expecting something like `overwriteWithPlatform`, and passing the props down to `useHostingProviderName`, `useHostingProviderURL` and `useHostingProviderURL` so we don't need to touch this level too much. But what you proposed is easier, so I'm fine with this, but we might want to reconsider the prop naming here.